README.md

```diff
@@ -77,6 +77,18 @@ docker run --detach --rm -p 9933:9933 -p 9944:9944 -p 9615:9615 \
 
 ## Run step by step
 
+To run a background worker to collect on-chain events, an instance of rabbitmq is required:
+
+```
+docker run -d -p 5672:5672 rabbitmq
```

Comment:
This command now repeats twice (line 112), I would rather create a link, something along the lines: "If you want to run a background worker .... see [event processing section] below. Alternatively you can disable ..."

But yeah, a refactor of this README is probably needed, a new stuff is just piling on and on.

Reply:
Yeah good point, I'll go ahead with your recommendation.  The reason I have suggested we make this explicit and mention it here is because after the [chains PR](https://github.com/paralink-network/paralink-node/pull/18) the node will error on startup if the background worker is enabled and there is no broker available.  This is because there will be 1 chain by [default](https://github.com/paralink-network/paralink-node/blob/master/src/data/default_chain_config.json), which subsequently triggers a celery task in the [collector](https://github.com/paralink-network/paralink-node/blob/9f6106c08cf1a59b88e47531b22ef82cb0c55905/src/process/collector.py#L19).  Previously this was not an issue when the collector [iterated over contract oracles ](https://github.com/paralink-network/paralink-node/blob/7c07c25cd0bdf3223837838819a9bf7cfa8eb6dc/src/process/collector.py#L10) as there were no contracts oracles by default and therefore no celery tasks created.